### ISSUE TITLE
Front camera illegal argument exception

### ISSUE THREAD

**Issue by HernanA4Q** [ketai/ketai#49]
Hello everyone.

I am using a Samsung J7. When I execute the setCameraID (1) method the following error appears:

java.lang.IllegalArgumentException: x + width must be <= bitmap.width() at android.graphics.Bitmap.checkPixelsAccess(Bitmap.java:1441) at android.graphics.Bitmap.setPixels(Bitmap.java:1514) at processing.core.PImage.loadPixels(Unknown Source) at ketai.camera.KetaiCamera.determineCameraParameters(KetaiCamera.java:1127) at ketai.camera.KetaiCamera.start(KetaiCamera.java:570) at processing.test.cameragettingstarted.CameraGettingStarted.mousePressed(CameraGettingStarted.java:72) at processing.core.PApplet.mousePressed(Unknown Source) at processing.core.PApplet.handleMouseEvent(Unknown Source) at processing.core.PApplet.dequeueEvents(Unknown Source) at processing.core.PApplet.handleDraw(Unknown Source) at processing.core.PGraphicsAndroid2D.requestDraw(Unknown Source) at processing.core.PApplet.run(Unknown Source) at java.lang.Thread.run(Thread.java:818) Exception caught while trying to connect to camera service. Please check your sketch permissions or that another application is not using the camera.

Search the Processing Forum and find the following entry:

https://forum.processing.org/two/discussion/17793/front-camera-on-android

In summary it says that the error is in the resize

public void setCameraID(int _id) {

if (_id < Camera.getNumberOfCameras())
cameraID = _id;

if (cameraID == 1) {
  int _temp = width;
  width = height;
  height = _temp;
  resize(width, height);//here error!!!!!!!!!!!!!!!!!!!!!!!!!!!!
  bitmap = Bitmap.createBitmap(pixels, width, height,
      Config.ARGB_8888);
}
}

Can you help me fix it?
Thank you

**Comment by ketai** [comment]
Removed offending code.  Tested and updated example.
